Presentation.Web/Controllers/API/V2/Internal/ItSystemUsages/ItSystemUsageMigrationV2Controller.cs

```diff
@@ -0,0 +1,110 @@
+using Swashbuckle.Swagger.Annotations;
+
+namespace Presentation.Web.Controllers.API.V2.Internal.ItSystemUsages
+{
+    [RoutePrefix("api/v2/internal/it-system-usages")]
+    public class ItSystemUsageMigrationV2Controller : InternalApiV2Controller
+    {
+        private readonly IItSystemUsageMigrationResponseMapper _responseMapper;
+        private readonly IItSystemUsageMigrationServiceAdapter _adapter;
+
+        public ItSystemUsageMigrationV2Controller(IItSystemUsageMigrationResponseMapper responseMapper,
+            IItSystemUsageMigrationServiceAdapter adapter)
+        {
+            _responseMapper = responseMapper;
+            _adapter = adapter;
+        }
+
+        [HttpGet]
+        [Route("{usageUuid}/migration")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(ItSystemUsageMigrationV2ResponseDTO))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public IHttpActionResult Get([Required][NonEmptyGuid] Guid usageUuid, [Required][NonEmptyGuid] Guid toSystemUuid)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            return _adapter.GetMigration(usageUuid, toSystemUuid)
+                .Select(_responseMapper.MapMigration)
+                .Match(Ok, FromOperationError);
+        }
+
+        [HttpPost]
+        [Route("{usageUuid}/migration")]
+        [SwaggerResponseRemoveDefaults]
+        [SwaggerResponse(HttpStatusCode.NoContent)]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public IHttpActionResult ExecuteMigration([Required][NonEmptyGuid] Guid usageUuid, [Required][NonEmptyGuid] Guid toSystemUuid)
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            return _adapter.ExecuteMigration(usageUuid, toSystemUuid)
+                .Match(NoContent, FromOperationError);
+        }
+
+        [HttpGet]
+        [Route("migration/permissions")]
+        [SwaggerResponse(HttpStatusCode.OK, Type = typeof(ItSystemUsageMigrationPermissionsResponseDTO))]
+        [SwaggerResponse(HttpStatusCode.BadRequest)]
+        [SwaggerResponse(HttpStatusCode.Forbidden)]
+        [SwaggerResponse(HttpStatusCode.Unauthorized)]
+        [SwaggerResponse(HttpStatusCode.NotFound)]
+        public IHttpActionResult GetPermissions()
+        {
+            if (!ModelState.IsValid)
+                return BadRequest(ModelState);
+
+            var permissions = _adapter.GetCommandPermissions();
+            return Ok(_responseMapper.MapCommandPermissions(permissions));
+        }
+
+        [HttpGet]
+        [Route("unused")]
```

Comment:
also this one only makes sense in the context of migration since it-system-usages/unused should always return nothing  if you had to believe the route :-)

.. lets move it to `migration/unused-it-systems`